dataflow-website/recipes/scaling/kubernetes/helm/prometheus/prometheus-configmap.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
```

Comment:
Same file in helm. Again, unclear what we are trying to do with this.

Reply:
Same as above